src/css/augmentedsteam.css

```diff
@@ -2202,10 +2202,19 @@ label[for="show_common_games"] {
   float: right;
 }
 
+.es_wsgf .details_block {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-top: -10px;
+  margin-right: -10px;
```

Comment:
These margins can be simplified to `gap: 10px` once support is wider: https://developer.mozilla.org/en-US/docs/Web/CSS/gap#support_in_flex_layout